src/main/java/ch/njol/skript/effects/EffPlaySound.java

```diff
@@ -55,7 +55,8 @@
 public class EffPlaySound extends Effect {
 
 	private static final boolean SOUND_CATEGORIES_EXIST = Skript.classExists("org.bukkit.SoundCategory");
-
+	private static final String SOUND_VALID_CHARACTERS = "[a-z0-9\\/._-]+"; // Minecraft only accepts these characters 

```

Comment:
Instead of having it as a String, I'd compile a Pattern here and then use a matcher when needed.